client/src/containers/UserInstrument.jsx

```diff
@@ -3,11 +3,19 @@ import React, { Component } from 'react';
 class userInstrument extends Component {
```

Comment:
Please rename this component to `UserInstrument`. Follows the filename and its local name when it's imported into another component.